compiler/rustc_middle/src/ty/query/profiling_support.rs

```diff
@@ -66,7 +67,7 @@ impl<'p, 'c, 'tcx> QueryKeyStringBuilder<'p, 'c, 'tcx> {
                 end_index = 3;
             }
             other => {
-                name = other.as_symbol();
+                name = Symbol::intern(&other.to_string());
```

Comment:
Similar situation here to the debuginfo one: the crate is less common, so it could take the `.to_string()` cost better than this would the `Symbol::intern` cost. And as always you can use variables to work around needing to do either.